### ISSUE TITLE
Maximum allowed artifact storage size of 75000 Mb will be exceeded.

### ISSUE THREAD

**Issue by julianxhokaxhiu** [appveyor/ci#3483]
As on #3417 it happened again:

https://ci.appveyor.com/project/julianxhokaxhiu/ffnx/builds/33844267

Can you please help me? I would appreciate if you could empower me somehow to do this cleanup on my own. I do delete my job history pretty often as I don't really care about uploaded artifacts on the job itself. Would be nice if upon a job history deletion you can also delete the relative artifacts.

Thank you in advance!

**Comment by julianxhokaxhiu** [comment]
I've read the documentation about Github deployments and I wonder if there's a way to re-create the `$artifacts` object within the powershell context, manually. Is there any docs about it?

I did try to use this to simulate the object data: https://www.appveyor.com/docs/packaging-artifacts/#getting-information-about-uploaded-artifacts

I did try to declare an objet within the `before_deploy` section like this (RELEASE_NAME and RELEASE_PATH points to real name and ZIP file paths within my manifest ):
```ps
$artifacts = @{}
$artifacts["RELEASE_NAME"] = @{
  name="RELEASE_NAME"
  path="RELEASE_PATH"
  type="zip"
}
```
And yet the Github deployer complained that was not able to find any artifact to be deployed, even if the RELEASE_NAME was in the `artifacts` property value.

Any hint in avoiding to upload the artifacts in AppVeyor and allowing me to deploy them directly to Github is appreciated.
